docs/scarpet/api/Entities.md

```diff
@@ -693,6 +717,34 @@ Applies status effect to the living entity. Takes several optional parameters, w
 `true` and `false`. If no duration is specified, or if it's null or 0, the effect is removed. If name is not specified,
 it clears all effects.
 
+### `modify(e, 'may_fly', boolean)`
+
+Sets the value of the player's `mayfly` (The ability of the player to fly) ability to the given boolean value.
+
+### `modify(e, 'flying', boolean)`
+
+Sets the value of the player's `flying` (If the player is flying or not) ability to the given boolean value.
+
+### `modify(e, 'may_build', boolean)`
+
+Sets the value of the player's `mayBuild` (The ability of the player to place blocks) ability to the given boolean value.
+
+### `modify(e, 'insta_build', boolean)`
+
+Sets the value of the player's `instabuild` (The ability of the player to place blocks without reducing their count in the inventory and shoot with a bow without arrows) ability to the given boolean value.
+
+### `modify(e, 'fly_speed', float)`
+
+Sets the value of the player's `flySpeed` (The speed at which the player moves while flying) ability to the given float value.
```

Comment:
Same thing about specific number types around here, if you really want say to the given number.